browser/ui/views/playlist/playlist_bubble.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_PLAYLIST_PLAYLIST_BUBBLE_H_
+#define BRAVE_BROWSER_UI_VIEWS_PLAYLIST_PLAYLIST_BUBBLE_H_
+
+#include "base/memory/weak_ptr.h"
+#include "brave/browser/ui/views/playlist/playlist_bubbles_controller.h"
+#include "ui/base/metadata/metadata_header_macros.h"
+#include "ui/views/bubble/bubble_dialog_delegate_view.h"
+
+namespace views {
+class View;
+class Widget;
+}  // namespace views
+
+namespace playlist {
+class PlaylistTabHelper;
+
+class PlaylistBubble : public views::BubbleDialogDelegateView {
```

Comment:
Totally optional: I think in `//chrome/ui`, we usually name toolkit independent interface as `FooBubble`(chrome/ui/foo_bubble) and views implementation as `FooBubbleView`(chrome/ui/views/foo_bubble_view). So at first glance, I thought like so. But it's still under `views` toolkit, PlaylistBubbleView might be better name.

Reply:
And the `ActionBubbleView` could be mouthful, I intended it to be paired with ActionIconView, so that we can infer that this bubble is somehow related to it.

Reply:
But these convention is not something written in stone, I'll leave it up to you.

Reply:
Okay, let's do `PlaylistBubbleView`, `PlaylistAddBubbleView`, and `PlaylistConfirmBubbleView`. This might also be the right time to rename our "confirm" bubble to better reflect its purpose. How about `PlaylistEditBubbleView`?

Reply:
Good call!